schema-engine/connectors/schema-connector/src/warnings.rs

```diff
@@ -12,11 +129,11 @@ pub struct Warnings {
     /// Fields that are using Prisma 1 CUID defaults.
     pub prisma_1_cuid_defaults: Vec<ModelAndField>,
     /// Fields having an empty name.
-    pub fields_with_empty_names_in_model: Vec<ModelAndField>,
+    pub fields_with_empty_names_in_model: GroupBy<ModelAndField>,
     /// Fields having an empty name.
-    pub fields_with_empty_names_in_view: Vec<ViewAndField>,
+    pub fields_with_empty_names_in_view: GroupBy<ViewAndField>,
     /// Fields having an empty name.
-    pub fields_with_empty_names_in_type: Vec<TypeAndField>,
+    pub fields_with_empty_names_in_type: GroupBy<TypeAndField>,
```

Comment:
These can stay `Vec<ModelAndField>` and we don't need the new `WriteOnlyVec` trait if we go with the previous suggestion (we'd just wrap in GroupBy at the last moment in `render_warnings_grouped`).

Reply:
Thank you, I have applied this.